frontend/src/components/ShootMaintenance/MaintenanceConstraintWarning.vue

```diff
@@ -0,0 +1,46 @@
+<!--
+SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <div v-if="!isMaintenancePreconditionSatisfied" :class="{ 'd-flex' : icon }">
```

Comment:
The `v-if="!isMaintenancePreconditionSatisfied"` should be outside of the component.

Reply:
flex box does vertical align the icon so that it matches the layout of other icons / texts in the same line